src/components/Tabs/Annotation/AnnotationImportButton.tsx

```diff
@@ -97,58 +81,68 @@ export default function AnnotationImportButton(props: AnnotationImportButtonProp
 
   const hasAnnotationData = annotationState.data.getLabels().length > 0;
 
-  const fileList: UploadFile[] = useMemo(() => {
-    if (!uploadedFile) {
-      return [];
-    }
-    return [
-      {
-        uid: "",
-        name: uploadedFile.name,
-        status: convertedAnnotationData !== null ? "done" : "error",
-      },
-    ];
-  }, [uploadedFile, convertedAnnotationData]);
```

Comment:
List of files is no longer shown, instead the Upload component is replaced with `AnnotationFileInfo`